api/labels.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"sort"
+	"time"
+
+	mapset "github.com/deckarep/golang-set"
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+// LabelsHandler is an http.Handler for the /api/labels endpoint.
+type LabelsHandler struct {
+	ctx context.Context
+}
+
+// apiLabelsHandler is responsible for emitting just the revision SHAs for test runs.
```

Comment:
Is this comment copypasta? It emits labels, I hope :)

Reply:
Sure was.